elasticsearch.yml.tpl

```diff
@@ -1,7 +1,7 @@
 cluster:
   name: ${cluster_name}
 
-network.host: _site_,_lo_
+network.host: "0.0.0.0"
```

Comment:
What interface is that? _site_ should suite internal interface automatically

Reply:
The thing is that the internal load balancer calls the service from the IP address which is not on compute instance. The load balancer calls the elasticsearch, it checks the interfaces, sees the IP is not local and denies the response. In this way, the response is sent to whatever IP address load balancer has.